db/schema.rb

```diff
@@ -373,6 +374,23 @@
     t.index ["updated_by_id"], name: "index_litigations_on_updated_by_id"
   end
 
+  create_table "locations", force: :cascade do |t|
```

Comment:
Hey @aabdaab what happened here? Worth to reset the database. I'm doing that all the time. That's why the need I stress of the need of good and complete seeds :P

Reply:
hmm, good question 🤔I ran migrations, let me reset

Reply:
I resetted the DB but same result; I guess develop is missing this schema bits

Reply:
Locations table is old, now it's geography.

Reply:
You have to reset table when beigin on develop, then go back to your branch, and run migrations. If you reset it on your branch that's gonna create those locations and sectors tables.